app/src/main/res/layout/journey_info_cell.xml

```diff
@@ -0,0 +1,104 @@
+                android:orientation="vertical">
+
+                <TextView
+                    android:id="@+id/depStation"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:text="depStation" />
+
+                <TextView
+                    android:id="@+id/depTime"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:text="depTime" />
+
+            </LinearLayout>
+
+            <TextView
+                android:id="@+id/arrow"
+                android:layout_width="wrap_content"
+                android:layout_height="match_parent"
+                android:layout_weight="1"
+                android:gravity="center"
+                android:text="->"
+                android:textAlignment="center" />
+
+            <LinearLayout
+                android:layout_width="0dp"
+                android:layout_height="match_parent"
+                android:layout_weight="4"
+                android:background="#B6AEAE"
+                android:gravity="center"
+                android:orientation="vertical">
+
+                <TextView
+                    android:id="@+id/arrStation"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:text="arrStation" />
+
+                <TextView
+                    android:id="@+id/arrTime"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:text="arrTime" />
+
+            </LinearLayout>
+
+            <TextView
+                android:id="@+id/price"
+                android:layout_width="0dp"
+                android:layout_height="match_parent"
+                android:layout_weight="2"
+                android:gravity="center"
+                android:text="£££"
+                android:textAlignment="center" />
+        </LinearLayout>
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal">
+
+            <TextView
+                android:id="@+id/carrier"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_weight="1"
+                android:background="#ED1919"
+                android:text="carrier" />
+
+            <TextView
+                android:id="@+id/journeyTime"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_weight="0.5"
+                android:background="#ED1919"
+                android:text="14:46"
+                android:textAlignment="textEnd" />
+        </LinearLayout>
+
+
+    </LinearLayout>
+</FrameLayout>
```

Comment:
Same comments as elsewhere on this xml - let's use the different xml files in the 'values' folder to pull a lot of this stuff out please